lib/linguist/languages.yml

```diff
@@ -3309,6 +3309,17 @@ ObjDump:
   tm_scope: objdump.x86asm
   ace_mode: assembly_x86
   language_id: 256
+ObjectScript:
+  type: programming
+  extensions:
+  - ".cls"
+  - ".inc"
+  - ".int"
```

Comment:
A very quick case-insensitive search on GitHub.com for [`extension:int routine`](extension:int ROUTINE) returns only 463 files, many of which wouldn't match your heuristic which means this file extension is almost certainly not popular enough for inclusion right now.

Reply:
Yes, `cls` right now is mostly used, but, `inc`, `int`, and `mac` still is a part of the language. 
In our language we don't need too much `inc` files, it could be just a one or some per project, as well as `mac` and `int`.

Reply:
They may be, however widespread usage across GitHub.com is required before the extension can be included in Linguist. This applies to all languages.